src/components/AutoCompleteTextarea/Textarea.js

```diff
@@ -671,7 +669,22 @@ class ReactTextareaAutocomplete extends React.Component {
               />
             )}
           </div>
-        )}
+        );
+      }
+      return null;
+    };
+
+    let { maxRows } = this.props;
```

Comment:
an old holdover i'm sure, but move this to where the other props are destructured